media-sound/helvum/helvum-0.4.1.ebuild

```diff
@@ -0,0 +1,162 @@
+# Copyright 2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+# Auto-Generated by cargo-ebuild 0.5.4
+
+EAPI=8
+
+CRATES="
+	aho-corasick-1.0.4
+	anyhow-1.0.75
+	autocfg-1.1.0
+	bindgen-0.66.1
+	bitflags-1.3.2
+	bitflags-2.4.0
+	cairo-rs-0.18.0
+	cairo-sys-rs-0.18.0
+	cc-1.0.82
+	cexpr-0.6.0
+	cfg-expr-0.15.4
+	cfg-if-1.0.0
+	clang-sys-1.6.1
+	convert_case-0.6.0
+	cookie-factory-0.3.2
+	equivalent-1.0.1
+	field-offset-0.3.6
+	futures-channel-0.3.28
+	futures-core-0.3.28
+	futures-executor-0.3.28
+	futures-io-0.3.28
+	futures-macro-0.3.28
+	futures-task-0.3.28
+	futures-util-0.3.28
+	gdk-pixbuf-0.18.0
+	gdk-pixbuf-sys-0.18.0
+	gdk4-0.7.2
+	gdk4-sys-0.7.2
+	gio-0.18.1
+	gio-sys-0.18.1
+	glib-0.18.1
+	glib-macros-0.18.0
+	glib-sys-0.18.1
+	glob-0.3.1
+	gobject-sys-0.18.0
+	graphene-rs-0.18.1
+	graphene-sys-0.18.1
+	gsk4-0.7.2
+	gsk4-sys-0.7.2
+	gtk4-0.7.2
+	gtk4-macros-0.7.2
+	gtk4-sys-0.7.2
+	hashbrown-0.14.0
+	heck-0.4.1
+	indexmap-2.0.0
+	lazy_static-1.4.0
+	lazycell-1.3.0
+	libc-0.2.147
+	libspa-0.7.0
+	libspa-sys-0.7.0
+	log-0.4.20
+	memchr-2.5.0
+	memoffset-0.7.1
+	memoffset-0.9.0
+	minimal-lexical-0.2.1
+	nix-0.26.2
+	nom-7.1.3
+	once_cell-1.18.0
+	pango-0.18.0
+	pango-sys-0.18.0
+	peeking_take_while-0.1.2
+	pin-project-lite-0.2.12
+	pin-utils-0.1.0
+	pipewire-0.7.0
+	pipewire-sys-0.7.0
+	pkg-config-0.3.27
+	proc-macro-crate-1.3.1
+	proc-macro-error-1.0.4
+	proc-macro-error-attr-1.0.4
+	proc-macro2-1.0.66
+	quote-1.0.33
+	regex-1.9.3
+	regex-automata-0.3.6
```

Comment:
`pkgcheck scan` should warn on this as `-` as a separator is deprecated; please use pycargoebuild.

Reply:
Yeah it did, I just did not want to bother at the time since I did not know about pycargoebuild :D